src/app/services/edition-data.service.ts

```diff
@@ -30,11 +32,57 @@ export class EditionDataService {
 
         return editionDoc;
       }),
+      mergeMap((editionData) => this.loadXIinclude(editionData, editionUrl.substring(0, editionUrl.lastIndexOf('/') + 1))),
       publishReplay(1),
```

Comment:
Why `publishReplay` instead of `shareReplay`?

Reply:
This was like this before, I've just added `mergeMap`. 
Anyway, `shareReplay` generates the following error which block the text from being shown.
<img width="522" alt="Schermata 2021-01-19 alle 10 27 00" src="https://user-images.githubusercontent.com/7116412/105014557-df8e8c00-5a40-11eb-9c65-a39863e4d5f4.png">

Reply:
If I use `shareReplay` then in the Network Tab **_n_** calls to xml files. They are all with code 304, so it should not be a problem, but I don't really like it, that's why originally was chosen `publishReplay(1)`, (in fact that is a part of code that was there before the current PR).